internal/backend/rclone/backend.go

```diff
@@ -183,7 +183,7 @@ func newBackend(ctx context.Context, cfg Config, lim limiter.Limiter) (*Backend,
 	dialCount := 0
 	tr := &http2.Transport{
 		AllowHTTP: true, // this is not really HTTP, just stdin/stdout
-		DialTLS: func(network, address string, _ *tls.Config) (net.Conn, error) {
+		DialTLSContext: func(_ context.Context, network, address string, _ *tls.Config) (net.Conn, error) {
```

Comment:
There's nothing to cancel here, so this can't break anything.

Reply:
Yes. I was looking at the flow and didn't initially realize it was unused.